AutomatedTesting/Gem/PythonTests/scripting/NodePalette_HappyPath_CanSelectNode.py

```diff
@@ -9,12 +9,14 @@
 from editor_python_test_tools.utils import Report
 import azlmbr.legacy.general as general
 import editor_python_test_tools.pyside_utils as pyside_utils
+import scripting_utils.scripting_constants as constants
```

Comment:
from scripting_utils.scripting import SCRIPT_CANVAS_UI, WAIT_TIME_3, NODE_PALETTE_QT

if you import the specific constants you use you won't have to use dot notation constants.A_CONSTANT and you limit the amount of stuff you bring in to just what you use. you can wrap the list of specific constants in parenthesis if you need to for line continuation

Reply:
nifty. I'll try that out

Reply:
Alternatively, if you're going for a more meaningful and readable codebase, it could be argued that the dot operator notation if you weren't using a generic name like `constants`. 

For example if you did the following:

`import scripting_utils.scripting_constants as SCWidgets`

You'd then be able to write one of the below lines that utilize it like so

`node_palette = sc.findChild(QtWidgets.QDockWidget, SCWidgets.NODE_PALETTE_QT)`

**General Note:** Avoid using overly generic names and terms while you develop things. As languages mature, new libraries are introduced to replace existing libraries that have been deprecated, and the life of the software persists,  it introduces risk that at some point those generic name usages could conflict with something newly introduce and cause unexpected behavior in tests/your program

Reply:
on inspection it looks like most of the test scripts in script canvas testing just need QtWidget and panel names. I added some more comments in the constants file to give them a bit more context. Also, I updated the import to be more specific per feedback